app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id 
+      redirect_to home_path, notice: 'You are signed in'
+    else
+      flash[:error] = 'Invalid email or password'
```

Comment:
You might need to change this to flash[:danger] to get it working with bootstrap 3.